src/load/azext_load/data_plane/load_trigger/custom.py

```diff
@@ -0,0 +1,233 @@
+import azext_load.data_plane.load_trigger.utils as utils
+
+logger = get_logger(__name__)
+
+def create_trigger_schedule(
+    cmd,
+    load_test_resource,
+    resource_group_name=None,
+    trigger_id=None,
+    description=None,
+    display_name=None,
+    trigger_start_date_time=None,
+    recurrence_type=None,
+    recurrence_interval=None,
+    recurrence_index=None,
+    recurrence_cron_expression=None,
+    recurrence_dates_in_month=None,
+    recurrence_week_days=None,
+    end_after_occurrence=None,
+    end_after_date_time=None,
+    test_ids=None,
+):
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    logger.info("Creating trigger schedule")
+    recurrence_end_body = utils.get_recurrence_end_body(
+        end_after_occurrence,
+        end_after_date_time,
+    )    
+    logger.debug("Recurrence end object: %s", recurrence_end_body)
+    recurrence_body = utils.get_recurrence_body(
+        recurrence_type,
+        recurrence_interval,
+        recurrence_index,
+        recurrence_cron_expression,
+        recurrence_dates_in_month,
+        recurrence_week_days,
+        recurrence_end_body,
+    )
+    logger.debug("Recurrence object: %s", recurrence_body)
+    trigger_body = models.ScheduleTestsTrigger(
+        test_ids=test_ids,
+        recurrence=recurrence_body,
+        start_date_time= trigger_start_date_time,
+        state=enums.TriggerState.ACTIVE,
+        display_name=display_name,
+        description=description,
+    )
+    logger.debug("Trigger schedule body: %s", trigger_body)
+    try:
+        response = client.create_or_update_trigger(trigger_id=trigger_id, body=trigger_body)
+        logger.debug("Created trigger schedule: %s", response)
+        logger.info("Creating trigger schedule completed")
+        return response
+    except Exception as e:
+        logger.error("Error occurred while creating trigger schedule: %s", str(e))
+
+
+def update_trigger_schedule(
+    cmd,
+    load_test_resource,
+    trigger_id,
+    resource_group_name=None,
+    description=None,
+    display_name=None,
+    trigger_start_date_time=None,
+    recurrence_type=None,
+    recurrence_interval=None,
+    recurrence_index=None,
+    recurrence_cron_expression=None,
+    recurrence_dates_in_month=None,
+    recurrence_week_days=None,
+    end_after_occurrence=None,
+    end_after_date_time=None,
+    test_ids=None,
+):
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    logger.info("Updating trigger schedule with name")
+    existing_trigger_schedule: models.ScheduleTestsTrigger = None
+    try:
+        existing_trigger_schedule = client.get_trigger(trigger_id)
+    except Exception as e:
+        logger.debug("Error occurred while fetching trigger schedule: %s", str(e))
+        logger.error("Trigger schedule with id: %s not found.", trigger_id)
+        return
```

Comment:
Here as well

Reply:
Here as well we should say 
Error occurred while fetching schedule trigger 
Schedule trigger with id